react/week2/my-app/src/addToDo.js

```diff
@@ -0,0 +1,26 @@
+//import ToDoArray from "./ToDoList";
+import ToDoRow from "./ToDoRow.js";
+
+export default function AddTodo(props) {
```

Comment:
I'd consider renaming this to something that indicates it's a list e.g. TodoList and then as you've done with the Row or TodoItem (but it's just for eligibility reasons no logic flaw!)

I originally thought this name was a function AddToDo